docs/kubernetes-shortguide-definitions/containerization-definition-shortguide/index.md

```diff
@@ -16,4 +15,4 @@ show_on_rss_feed: false
 
 ### Containerization
 
-A virtualization method to run distributed applications in containers using microservices. Containerizing an application requires a base image that can be used to create an instance of a container. Once an application’s image exists, you can push it to a centralized container registry that Kubernetes can use to deploy container instances in a cluster’s pods.
+Containerization is a software architecture practice that organizes applications and their dependencies in containers. Containerizing an application requires a base image that can be used to create an instance of a container. Once an application’s image exists, you can push it to a centralized container registry. Docker, Kubernetes, and other orchestration tools can download images from a registry to deploy container instances.
```

Comment:
Split the last sentence into two. I also think that containers don't really use microservices--instead, containers are a thing that enables you to create microservices (though you could still create a monolithic app with containers)